core/dbt/parser/unit_tests.py

```diff
@@ -203,6 +206,22 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock) -> None:
         self.schema_parser = schema_parser
         self.yaml = yaml
 
+    def load_rows_from_seed(self, seed_name):
+        rows = []
+
+        try:
+            seed_node = self.manifest.ref_lookup.perform_lookup(
+                f"seed.{self.project.project_name}.{seed_name}", self.manifest
+            )
+            seed_path = Path(seed_node.root_path) / seed_node.original_file_path
+            with open(seed_path, "r") as f:
+                for row in DictReader(f):
+                    rows.append(row)
+        except DbtInternalError:
```

Comment:
can we catch this further up - after `self.manifest.ref_lookup`? that way the error handling remains localized and precise